docs/markdown/Reference-tables.md

```diff
@@ -4,26 +4,26 @@
 
 These are return values of the `get_id` method in a compiler object.
```

Comment:
The sentence prefacing the table doesn't make sense after you've added a column.  Do you really want to say 'other' when the value of get_argument_syntax is undefined.  Would it make more sense to have a separate table for this?

Reply:
I tried a separate table, I didn't like that the keys of the table were copied between the two tables. I've removed "other" now undefined values are just left blank.